web/styles/informational_overlays.css

```diff
@@ -1,108 +1,186 @@
 .informational-overlays {
+    display: grid;
+    place-items: center;
+
     .overlay-content {
-        /* because zoom breaks at 525px perhaps due to rounding errors, so add a
-           trivial amount of width so it doesn't break. */
-        width: 550px;
+        width: 90%;
+        max-width: 650px;
         margin: 0 auto;
         position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
+        top: unset;
+        border-radius: 6px;
         overflow: hidden;
 
         background-color: var(--color-background-modal);
     }
 
     .overlay-tabs {
-        padding: 10px 0;
-        border-bottom: 1px solid hsl(0deg 0% 0% / 20%);
+        position: relative;
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        gap: 15px;
+        padding: 0 5%;
+        height: 60px;
+        box-shadow: 0 3px 12px 1px hsl(207deg 100% 11% / 8%);
+        z-index: 2;
 
         .tab-switcher {
-            margin-left: 15px;
+            flex-grow: 1;
         }
 
         .exit {
-            float: right;
-            font-size: 1.5rem;
-            color: hsl(0deg 0% 67%);
-            font-weight: 600;
-            margin: 0 15px;
+            display: inline-block;
+            font-size: 16px;
+            line-height: 1em;
+            text-align: center;
+            width: 1em;
+            color: hsl(0deg 0% 60%);
             cursor: pointer;
+            order: 1;
         }
     }
 
-    .overlay-modal {
-        padding-bottom: 10px;
+    .modal-body {
+        height: 75vh;
+        max-height: unset;
+        text-align: center;
+        outline: none;
+    }
+
+    .simplebar-content {
+        padding: 35px 30px 35px 25px !important;
+    }
+
+    & a {
+        color: var(--color-text-link);
 
-        .modal-header h3 {
-            font-weight: 300;
+        &:hover {
+            color: var(--color-text-link) !important;
         }
+    }
 
-        .modal-body {
-            height: 70vh;
-            text-align: center;
-            outline: none;
+    .documentation-link {
+        text-align: center;
+        margin-top: 35px;
+    }
+
+    & kbd {
+        font-family: "Source Sans 3", sans-serif;
+        font-feature-settings: "ss01" on;
```

Comment:
Be sure to leave a comment about what `font-feature-settings` you're enabling here. `ss01` likely refers to some stylistic alternates, so it would be good to know which ones :)

Reply:
This comment was added by @Taiki-Choda in the original PR. I did some research and discovered that it provides alternate glyphs for certain characters (different styling/variants).

Upon comparing the differences between using the `font-feature-settings` property and not using it, I noticed that it only impacts the character I, showcasing a distinct variant:

Without `font-feature-settings`: 
![image](https://github.com/zulip/zulip/assets/64723994/9138a34e-f9a9-43e4-bb08-0d25f17108b4)

With font-feature-settings::
![image](https://github.com/zulip/zulip/assets/64723994/09212faa-f18f-4981-a523-c0e53bb2404f)

The `I` with `font-feature-settings: "ss01" on;` was looking better so I kept it :)

Reply:
Ah, that's awesome. That alternate glyph makes a big difference (and I agree the version with the bars on the I should be kept), so it'll be very useful to have the comment explaining this in the code.

Reply:
Added the same content that was in tooltips.css for hotkeys hints:
![image](https://github.com/zulip/zulip/assets/64723994/0df35c26-7bb0-4dba-81e2-3cf4d050df87)